SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -167,6 +168,9 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	bool bWaitingForAcceptingPlayersToSpawn;
 	FString SnapshotToLoad;
 
+	uint32 ActorReplicationRateLimit = GetDefault<USpatialGDKSettings>()->ActorReplicationRateLimit;
```

Comment:
Let's not do this here and instead initialize in the constructor. Even if this does the same, it's not obvious (and I didn't manage to find out whether this would be guaranteed to evaluate this every time when creating a `SpatialNetDriver`).